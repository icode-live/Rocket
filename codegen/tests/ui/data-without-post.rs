#![feature(plugin, decl_macro)]
#![plugin(rocket_codegen)]

extern crate rocket;

#[get("/", data = "<something>")]
fn get(something: rocket::Data) -> &'static str { "hi" }
