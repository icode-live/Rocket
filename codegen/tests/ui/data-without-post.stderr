error: `data` route parameters can only be used with payload supporting methods
 --> $DIR/data-without-post.rs:6:12
  |
6 | #[get("/", data = "<something>")]
  |            ^^^^^^^^^^^^^^^^^^^^
  |
  = note: 'GET' does not support payloads

error: aborting due to previous error

